site/content/3.12/release-notes/version-3.12/whats-new-in-3-12.md

```diff
@@ -241,5 +241,59 @@ of outgrowing the maximum number of file descriptors the ArangoDB process
 can open. Thus, these options should only be enabled on deployments with a
 limited number of collections/shards/indexes.
 
+## Client tools
+
+### arangodump
+
+#### Improved dump performance
+
+ArangoDB 3.12 includes extended parallelization capabilities to work not only
+at the collection level, but also at the shard level. In combination with the
+new optimized format, database dumps are now created and restored quickly and
+occupy minimal disk space. This major performance boost makes dumps five times
+faster and restores three times faster, which is extremely useful when dealing
+with large shards.
+
+#### Resource usage limits
+
+The following startup options that can be used to limit
+the resource usage of parallel _arangodump_ invocations have been added:
```

Comment:
It should be mentioned here explicitly that these are startup options for arangod, not for arangodump.